AGPs/AGP-ANSP-X.md

```diff
@@ -0,0 +1,64 @@
+---
+AGP: N/A
+Title: Retainer for Ongoing Aragon Network Security Provider
+Author: maurelian
+Status: Stage III
+Track: Finance
+Created: 2019-04-10
+---
+
+### Summary
+
+We propose to continue working with the Aragon One team on a retainer basis for a 1 year period. 
```

Comment:
We propose this to be an engagement to working with the entities responsible of developing the code of interest, without mentioning a specific Flock team.

The agreement will be entered into with the Aragon Association anyways, and as explained further down in the proposal, the budget could be allocated to other Flock teams.

(Same goes with the rest of the proposal)

Reply:
The updated summary now says:
> We propose to continue our work securing the Aragon Network, on a retainer basis for a 1 year period, working with the entities responsible for developing AragonOS, Aragon Apps, and other code of interest.